gems/pending/VMwareWebService/MiqVimBroker.rb

```diff
@@ -527,6 +527,8 @@ def connectionInfo
   end
 
   def logStatus
+    return unless $vim_log.info?
+
```

Comment:
@chessbyte is right, this needs to come after the client check - in the server portion of the code. Nice catch.